src/Condition.php

```diff
@@ -0,0 +1,268 @@
+<?php
+
+namespace Nahid\JsonQ;
+
+class Condition
+{
+    /**
+     * Simple equals
+     *
+     * @param mixed $value
+     * @param mixed $comparable
+     *
+     * @return bool
+     */
+    public static function equal($value, $comparable)
+    {
+        return $value == $comparable;
+    }
+
+    /**
+     * Strict equals
+     *
+     * @param mixed $value
+     * @param mixed $comparable
+     *
+     * @return bool
+     */
+    public static function strictEqual($value, $comparable)
+    {
+        return $value === $comparable;
+    }
+
+    /**
+     * Simple not equal
+     *
+     * @param mixed $value
+     * @param mixed $comparable
+     *
+     * @return bool
+     */
+    public static function notEqual($value, $comparable)
+    {
+        return $value != $comparable;
+    }
+
+    /**
+     * Strict not equal
+     *
+     * @param mixed $value
+     * @param mixed $comparable
+     *
+     * @return bool
+     */
+    public static function notExactEqual($value, $comparable)
+    {
+        return $value !== $comparable;
+    }
+
+    /**
+     * Strict greater than
+     *
+     * @param mixed $value
+     * @param mixed $comparable
+     *
+     * @return bool
+     */
+    public static function greater($value, $comparable)
```

Comment:
Why not named the method ```greaterThan````?

Reply:
I think, you are right about all cases. Thanks.